src/measure_theory/function/uniform_integrable.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kexing Ying
 -/
 import measure_theory.function.convergence_in_measure
+import measure_theory.function.conditional_expectation
```

Comment:
Since uniform integrability is a quite low level notion, I'd rather not import conditional_expectation here. You could put that lemma in the conditional expectation file.

An idea for the near future: we should cut the conditional expectation file in two, with one file for the construction of condexp and basic api lemmas and another file for its properties.